### PR TITLE
(CAT-1346) erb_to_epp conversion for mod directory

### PR DESCRIPTION
**Pull request by praj1001**
## Summary
Conversion of ERB to EPP templates for puppetlabs-apache module. The files considered for conversion only limits to being present in the mod directory.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

**Review by Ramesh7 — APPROVED** [review]

**Comment by Ramesh7** [comment]
@praj1001 can you please rebase your PR?

File: manifests/mod/setenvif.pp [hunk 1]
```diff
@@ -11,7 +11,7 @@
     ensure  => file,
     path    => "${apache::mod_dir}/setenvif.conf",
     mode    => $apache::file_mode,
-    content => template('apache/mod/setenvif.conf.erb'),
+    content => epp('apache/mod/setenvif.conf.epp'),
```
Comment (Ramesh7) [inline manifests/mod/setenvif.pp:14]: this template doesn't have any params, is it expected?

File: templates/mod/_allow.epp [hunk 2]
```diff
@@ -0,0 +1,7 @@
+    Order deny,allow
+    Deny from all
+<% if $allow_from and !$allow_from.empty { -%>
+    Allow from <%= Array($allow_from).join(" ") %>
+<% }else { -%>
```
Comment (Ramesh7) [inline templates/mod/_allow.epp:5]: maint - 
```
<% } else { -%>
```

File: templates/mod/_require.epp [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<% $_requires = if $requires { %>$requires<% }else {%>$requires_defaults<%} %>
```
Comment (Ramesh7) [inline templates/mod/_require.epp:1]: can we make sure consistent syntax across the temples?

File: manifests/mod/setenvif.pp (hunk 1 above)
Comment (praj1001) [reply manifests/mod/setenvif.pp:14]: Yes, it doesnt have any params. This is expected behaviour

File: templates/mod/_require.epp (hunk 3 above)
Comment (praj1001) [reply templates/mod/_require.epp]: Sure

File: templates/mod/_allow.epp (hunk 2 above)
Comment (praj1001) [reply templates/mod/_allow.epp]: Done

**Review by dinesh-gujar — APPROVED** [review]
Nice Work

**Review by Ramesh7 — APPROVED** [review]

**Pull request merged**